2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -3,33 +3,9 @@ title: "R Notebook"
 output: html_notebook
 ---
 
-This is an [R Markdown](http://rmarkdown.rstudio.com) Notebook. When you execute code within the notebook, the results appear beneath the code. 
-
-Try executing this chunk by clicking the *Run* button within the chunk or by placing your cursor inside it and pressing *Cmd+Shift+Enter*. 
-
-```{r}
-remotes::install_github("MRCToxBioinformatics/Proteomics_data_analysis", dependencies='Suggests')
-
-remotes::install_github("MRCToxBioinformatics/Proteomics.analysis.data")
-
-if (!require("BiocManager", quietly = TRUE))
-    install.packages("BiocManager")
-BiocManager::install(version = "3.16")
-
-BiocManager::install("preprocessCore")
-
-
-install.packages("remotes")
-
-remotes::install_github("CambridgeCentreForProteomics/camprotR", dependencies = TRUE)
-
-remotes::install_github("CambridgeCentreForProteomics/camprotR")
-#remove.packages("camprotR")
-
-BiocManager::install("MSnbase")
-
-install.packages("here")
+ADD EXPLANATION WAS THE NOTEBOOK IS FOR. E.G AIMS, STEPS, ETC
```

Comment:
Always good to start with an explanation what the notebook is about. You'll need it when you come back in the future and helps to ensure you keep notebooks within a defined remit.